force-app/main/default/triggers/InsuranceCheck.trigger

```diff
@@ -0,0 +1,31 @@
+trigger InsuranceCheck on Medical_Appointment__c (after insert,after update) {
```

Comment:
pierwsz i najważniejsze nie widzę tutaj wzorca trigger handlera chociaż zadanko niby już jest na done

Reply:
poza tym wygląda git

Reply:
a no i build Ci nie przechodzi

Reply:
trigger handler się pojawi jak będzie ostatni trigger, ale widzę, że już jest to zaraz wrzucę

Reply:
już możesz dodać kod do trigger handera